src/software/utils/sfmColorHarmonize/colorHarmonizeEngineGlobal.hpp

```diff
@@ -16,13 +16,26 @@
 
 namespace aliceVision {
 
-enum EHistogramSelectionMethod
+enum class EHistogramSelectionMethod
 {
-    eHistogramHarmonizeFullFrame     = 0,
-    eHistogramHarmonizeMatchedPoints = 1,
-    eHistogramHarmonizeVLDSegment    = 2,
+    eHistogramHarmonizeFullFrame = 0,
+    eHistogramHarmonizeMatchedPoints,
+    eHistogramHarmonizeVLDSegment
 };
 
+inline std::string EHistogramSelectionMethod_description()
+{
+  return "Histogram selection method: \n"
+         "* full frame \n"
```

Comment:
Need the real valid values for that option, people cannot invent that it needs an underscore (like "full_frame").

Reply:
Fixed :)